browser/components/MarkdownEditor.js

```diff
@@ -8,6 +8,10 @@ class MarkdownEditor extends React.Component {
   constructor (props) {
     super(props)
 
+    this.hotkey = props.config.hotkey
+
+    this.keyPressed = []
```

Comment:
I think using `state` better, too.
Simply, use `state` for keeping state of the component :)
You can add state by writing blow.

```javascript
this.state = {
  // Other states...
  keyPressed: []
}
```

If you want to update the state, use `setState` method.

```javascript
this.setState({
  keyPressed: [a, b, c]
})
```

If you want to write smartly, you should use the logic of Redux. But now, we can't take advantage of Redux. 
So, you don't need using logic of Redux.